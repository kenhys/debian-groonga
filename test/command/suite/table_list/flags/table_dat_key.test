table_create bookmarks TABLE_DAT_KEY ShortText
table_list
