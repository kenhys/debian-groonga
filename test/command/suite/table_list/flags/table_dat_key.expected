table_create bookmarks TABLE_DAT_KEY ShortText
[[0,0.0,0.0],true]
table_list
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        "id",
        "UInt32"
      ],
      [
        "name",
        "ShortText"
      ],
      [
        "path",
        "ShortText"
      ],
      [
        "flags",
        "ShortText"
      ],
      [
        "domain",
        "ShortText"
      ],
      [
        "range",
        "ShortText"
      ],
      [
        "default_tokenizer",
        "ShortText"
      ],
      [
        "normalizer",
        "ShortText"
      ]
    ],
    [
      256,
      "bookmarks",
      "db/db.0000100",
      "TABLE_DAT_KEY|PERSISTENT",
      "ShortText",
      null,
      null,
      null
    ]
  ]
]
